src/components/Tabs.js

```diff
@@ -63,8 +63,8 @@ export default class Tabs extends Component {
     }
 
     const state = {
-      // Set focus if the change was triggered from the keyboard
-      focus: event.type === 'keydown',
+      // Sets focus to the active tab (should this always be true?)
+      focus: event.type === 'keydown' || event.type === 'click',
```

Comment:
Yeah lets set it to true I guess. I cannot remember why we only did this for keyboard. Probably because We did not want the outline to show instantly when clicking the tabs. 
I consider this breaking, so I will release this as major version and change the css to hide the focus outline.
Sounds okay?

Reply:
I think always setting it to true makes it more confusing, actually.  I think it turns the variable into something like `hasTabBeenSelected: boolean`, where it's false until it's true.  UncontrolledTabs would then always pass `focus: true` to the children.  It may make more sense to actually add an onBlur handler to set `focus: false`.

Reply:
Was just following up if you had any further movement on this fix?

Reply:
what else needs fixing?

Reply:
maybe I dont know how your release process works for PR's but if this is ready to go doesnt this need to get merged and released to a new version?